spec/features/course/assessment/question/text_response_management_spec.rb

```diff
@@ -145,7 +145,7 @@
         visit course_assessment_path(course, assessment)
 
         within find('section', text: question.title) { click_button 'Delete' }
-        click_button 'Delete question'
+        accept_prompt
```

Comment:
```
Failure/Error: super

ArgumentError:
  Unused parameters passed to Capybara::Queries::SelectorQuery : [{:visible=>false}]
./spec/features/course/assessment/question/text_response_management_spec.rb:147:in `block (5 levels) in <top (required)>'
------------------
--- Caused by: ---
Capybara::ElementNotFound:
  Unable to find button "Delete" that is not disabled
  ./spec/features/course/assessment/question/text_response_management_spec.rb:147:in `block (5 levels) in <top (required)>'
```
Flaky issue lies in line 147 not 148.

Reply:
both lines 147 and 148 are actually flaky. `check_button` does not wait for the button to exist before attempting to click.

In this regards, I changed both `click_button` to `find`, so that the test will wait for each button (Delete and also Delete Question) to appear on page first before attempting to click.

> Let's avoid making extra commits unless we know what's the cause of the flakiness, unless you're just testing out, then move PR to draft.

Roger on that